com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -338,6 +349,7 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
             #pragma multi_compile_fragment _ _DEFERRED_ADDITIONAL_LIGHT_SHADOWS
             #pragma multi_compile_fragment _ _SHADOWS_SOFT
             #pragma multi_compile_fragment _ _GBUFFER_NORMALS_OCT
+            #pragma multi_compile_fragment _ _CLEARCOAT // _CLEARCOATMAP
```

Comment:
If we only set this from material and not as a global keyword then the _CLEARCOATMAP should be included even if we don't support the map for clear coat. _CLEARCOAT and _CLARCOATMAP have been set up as independent keywords (like enum, not combinatorial bools).